### PR TITLE
Post 0.5.0 updates

### LINKED ISSUE #516: visual regression - split line on xaxis is visible
screen shot 2018-04-28 at 22 01 03

### LINKED ISSUE #518: No description on pypi site
screen shot 2018-04-28 at 22 12 27

### PR DESCRIPTION
**Pull request by chfw**
- add description on pypi site, fix #518
- remove x split line, fix #516

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,17 +16,24 @@
...
+    """Read a file into setup"""
+    with codecs.open(afile, 'r', 'utf-8') as opened_file:
+        return opened_file.read()
+
+
 setup(
     name=__title__,
     version=about["__version__"],
     description=__description__,
+    long_description=read("README.md"),
```
Comment (kinegratii) [inline setup.py]: This does not work.Markdown support is a new feature for PyPi in last two month.You may follow [this article](https://dustingram.com/articles/2018/03/16/markdown-descriptions-on-pypi) now.

Comment (kinegratii) [reply setup.py]: *README.md* is too long. It is better use another *long_description.md* file.

**Comment by chfw** [comment]
@chenjiandongx , this is your task 🥇 

3. Upgrade your setuptools:
You’ll need a version of setuptools>=38.6.0 to be able to produce a distribution with the new metadata.

$ pip install -U setuptools
You also must be actually using setuptools. This means that in your setup.py you have an import like:

from setuptools import setup
4. Make a source distribution
This is the same as usual:

$ python setup.py sdist
You may see a warning from distutils which can be safely ignored:

Unknown distribution option: 'long_description_content_type'
Note: If you’d like to create a wheel as well, perhaps with python setup.py bdist_wheel, you will need a version of wheel>=0.31.0:

$ pip install -U wheel
5. Upgrade your twine:
You must upload the distribution with twine, and only twine>=1.11.0 will correctly send the metadata to PyPI:

$ pip install -U twine
6. Upload the distribution:
This is also the same as usual:

$ twine upload dist/*

File: setup.py (hunk 1 above)
Comment (chfw) [reply setup.py]: not long enough! :), I have not included change log inside it yet.

**Pull request merged**